docs/JLBP-8.md

```diff
@@ -6,8 +6,8 @@ annotations if they live in a stable library, as outlined in
 used, they can cause problems when there are surface breakages (which is allowed
 per the rules of unstable functionality, explained in [JLBP-3](JLBP-3.md)). If
 such breakages can cause widespread pain, the unstable features can become
-"common-law GA." The preferable path is to promote such widely-used unstable
-libraries and features to stability, even if the functionality is not complete.
+"common-law GA." Promote widely-used unstable libraries and features to 1.0 
```

Comment:
> the unstable features can become "common-law GA."

I was looking for a way to replace the word "common-law" with something more plain and simple. How about "people may unexpectedly recognize the unstable features as GA." ?

Reply:
How about "de facto"?